src/routes/components/Accordion.tsx

```diff
@@ -140,6 +147,56 @@ export default function AccordionPage() {
               `}
             />
 
+            <CodeSnippet
+              lang="typescript"
+              tags="angular"
+              allowCopy={true}
+              code={`
+                <goa-button type="tertiary" mb="xl" (_click)="onClick($event)">
+                  Toggle Accordion Group
+                </goa-button>
+                
+                <goa-accordion heading="Heading" open={open}>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                </goa-accordion>
+                <goa-accordion heading="Heading" open={open}>
+                  <goa-badge type="success" content="Success" slot="headingcontent"></goa-badge>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                </goa-accordion>
+                <goa-accordion heading="Heading" open={open}>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                </goa-accordion>
+              `}
+            />
+
+            <CodeSnippet
+              lang="typescript"
+              tags="react"
+              allowCopy={true}
+              code={`
+                const [open, setOpen] = useState<boolean>(false);
+                const onClick = () => setOpen(!open);
+                
+                <GoAButton type="tertiary" mb="xl" onClick={onClick}>
```

Comment:
You will need to move this to it's own CodeSnippet with a `lang` of `tsx`

Reply:
This is what renders now
![image](https://github.com/GovAlta/ui-components-docs/assets/41582/a2d36151-1660-4e5f-b59c-b42963f2f7ce)